SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DonationTypes.swift

```diff
@@ -24,8 +24,26 @@
 
 import Foundation
 
+// Donation Type Data
+//{
+//    "data": [
+//        {
+//            "id": 1,
+//            "type": "Victims"
+//        },
+//        {
+//            "id": 2,
+//            "type": "Protesters"
+//        },
+//        {
+//            "id": 3,
+//            "type": "Movement"
+//        }
+//    ]
+//}
+
```

Comment:
@dasautoooo Can you remove these comments if they don't represent anything

Reply:
I added this intentionally because I wanna make sure other people know why this should start from 1.